install_files/ansible-base/ansible.cfg

```diff
@@ -13,5 +13,5 @@ agnostic_become_prompt=False
 
 [ssh_connection]
 scp_if_ssh=True
-ssh_args = -o ControlMaster=auto -o ControlPersist=1200
+ssh_args = -o ControlMaster=auto -o ControlPersist=1200 -o ServerAliveInterval=10 -o ServerAliveCountMax=3
```

Comment:
According to `man ssh_config`, the `ServerAliveCountMax` option defaults to 3, but no complaints about explicitly redeclaring that in the settings.

Reply:
Yep, it was just to ensure the maximum time we'd wait for a response.